xarray/plot/utils.py

```diff
@@ -360,15 +360,37 @@ def _infer_xy_labels(darray, x, y, imshow=False, rgb=None):
             raise ValueError("DataArray must be 2d")
         y, x = darray.dims
     elif x is None:
-        if y not in darray.dims and y not in darray.coords:
+        if (
+            y not in darray.dims
+            and y not in darray.coords
+            and y not in darray._level_coords
+        ):
             raise ValueError("y must be a dimension name if x is not supplied")
         x = darray.dims[0] if y == darray.dims[1] else darray.dims[1]
     elif y is None:
-        if x not in darray.dims and x not in darray.coords:
+        if (
+            x not in darray.dims
+            and x not in darray.coords
+            and x not in darray._level_coords
+        ):
             raise ValueError("x must be a dimension name if y is not supplied")
         y = darray.dims[0] if x == darray.dims[1] else darray.dims[1]
-    elif any(k not in darray.coords and k not in darray.dims for k in (x, y)):
-        raise ValueError("x and y must be coordinate variables")
+    else:
+        if any(
+            k not in darray.coords
+            and k not in darray.dims
+            and k not in darray._level_coords
+            for k in (x, y)
+        ):
+            raise ValueError("x and y must be coordinate variables")
```

Comment:
Again, technically it is a coordinate variable, dimension or MultiIndex level but this is super verbose - suggestions? Or just leave it as it is?